atlasdb-jepsen-tests/src/test/java/com/palantir/atlasdb/jepsen/events/EventTest.java

```diff
@@ -29,7 +28,9 @@
 import clojure.lang.Keyword;
 
 public class EventTest {
```

Comment:
I was trying not to add too much in here: but I'm wondering if we should split up this test class into `OkEventTest`, `FailEventTest`, etc. It's getting a bit long.